README.md

```diff
@@ -143,3 +143,37 @@ search.addWidget(
 );
 ```
 
+### multipleChoiceList
+
+#### API
```

Comment:
I personally never use jsdoc and I think having a nice README and examples is always preferred but happy to write a few lines of jsdoc